docs/guides/api-keys.md

```diff
@@ -0,0 +1,55 @@
+# Setting provider API keys
+
+Providers may require authentication details in order to use their APIs, typically API keys or tokens. By default Superface OneSDK reads API keys from environment variables. This behavior can be modified through the `super.json` file.
+
+## Getting API keys from provider
+
+Every provider has different way of obtaining keys, usually you need to sign up for an account. The documentation covers tutorials for several providers. In case you have trouble obtaining credentials from your selected provider, [reach to us](../support.mdx).
```

Comment:
I would ditch the sentence about tutorials for getting provider keys as we have only four and don't really plan expanding that number any time soon.